README.md

```diff
@@ -119,7 +119,7 @@ If upgrading older playbooks which were built prior to Ansible 2.10 and this col
 
   tasks:
     - name: Create ssh key
-      digital_ocean_sshkey:
+      sshkey:
```

Comment:
According to the text before this example playbook, this should **not** be changed.